crates/router/src/services/connector_integration_interface.rs

```diff
@@ -439,71 +439,6 @@ impl api::ConnectorCommon for ConnectorEnum {
     }
 }
 
-impl<T, ResourceCommonData, Req, Resp> api::ConnectorCommon
-    for ConnectorIntegrationEnum<'_, T, ResourceCommonData, Req, Resp>
```

Comment:
Do we still need `ConnectorIntegrationEnum` or can we remove it completely?

Reply:
We can't remove it yet. This is needed while we migrate the trait implementation connector by connector.